### DISCUSSION TITLE
Cluster Import error

### DISCUSSION THREAD

**Discussion by naveeddil** [xlg8/APMIS-Project#664] [General]
The process is not completed nor does giving any error report. If override is checked that means all previous cluster data should be replaced but I think that is the issue here.

@RaziaMahmodi @bnygren

![APMIS Cluster error](https://github.com/AFG-Polio-Data/APMIS-Project/assets/54671296/0f1414c3-32cc-456f-a83c-96073313ae52)

**Comment by RaziaMahmodi** [comment]
@naveeddil Dev team is working on it.
